api/txspool.go

```diff
@@ -279,5 +279,10 @@ func (a *API) verifyPoolL2TxWrite(txw *l2db.PoolL2TxWrite) error {
 			))
 		}
 	}
+	// Extra sanity checks: those checks are valid as per the protocol, but are very likely to
+	// have unexpected side effects that could have a negative impact on users
+	if poolTx.Type == common.TxTypeExit && poolTx.Amount.Cmp(big.NewInt(0)) <= 0 {
+		return tracerr.New(ErrExitAmount0)
+	}
```

Comment:
_Only a suggestions, it's up to you to accept _
```suggestion
		case common.TxTypeExit:
		// Extra sanity checks: those checks are valid as per the protocol, but are very likely to
		// have unexpected side effects that could have a negative impact on users
		if poolTx.Amount.Cmp(big.NewInt(0)) <= 0 {
			return tracerr.New(ErrExitAmount0)
		}
	}
```